admin/form-builder/assets/js/form-builder.js

```diff
@@ -142,6 +143,38 @@
                 // bring newly added element into viewport
                 Vue.nextTick(function () {
                     var el = $('#form-preview-stage .wpuf-form .field-items').eq(payload.toIndex);
+                    if ('yes' == payload.field.is_meta && state.show_custom_field_tooltip) {
+
+                        var image_one  = wpuf_assets_url + '/images/custom-fields/settings.png';
+                        var image_two  = wpuf_assets_url + '/images/custom-fields/advance.png';
+                        var html       = '<div class="wpuf-custom-field-instruction">';
+                            html      += '<div class="step-one">';
+                            html      += '<p style="font-weight: 400">Navigate through<strong><code>WP-admin > WPUF > Settings > Frontend Posting</code></strong>- there you have to check the checkbox: "Show custom field data in the post content area"</p>';
+                            html      += '<img src="'+ image_one +'" alt="settings">';
```

Comment:
These texts need to be translated, can't be static.

Reply:
> These texts need to be translated, can't be static.

This has been updated.